importers/src/metadata/index.ts

```diff
@@ -0,0 +1,344 @@
+export const enBookMetadata = new Map([
```

Comment:
general question: where is this metadata used? Book names and orders are not consistent within translations of one language. is this some kind of fallback in case other metadata is not available during import?

Reply:
Yeah, this is used in importStepLibrary.ts, for ESV, SBLGNT, and NASB

Just a fallback in case not specified, gives people a starting place